packages/eslint-plugin/rules/padding-line-between-statements/padding-line-between-statements._js_.test.ts

```diff
@@ -2736,6 +2736,72 @@ run({
       ],
       parserOptions: { ecmaVersion: 2022 },
     },
+
+    // ----------------------------------------------------------------------
+    // multiline-export
```

Comment:
these tests are similar to the multiline-var and multiline-const ones above. I have copied them and tweaked them to  better test the export case.